packages/suite/src/views/wallet/coinmarket/common/CoinmarketProviderInfo.tsx

```diff
@@ -19,19 +21,18 @@ export const CoinmarketProviderInfo = ({ exchange, providers }: CoinmarketProvid
 
     return (
         <Row data-testid="@coinmarket/form/info/provider" gap={spacings.xs}>
-            {!exchange && 'Unknown provider'}
-            {!provider && exchange}
-            {provider && (
+            {!exchange && <Translation id="TR_COINMARKET_UNKNOWN_PROVIDER" />}
+            {provider ? (
                 <>
                     {provider.logo && (
-                        <img
-                            width="16px"
-                            src={invityAPI.getProviderLogoUrl(provider.logo)}
-                            alt=""
-                        />
+                        <CoinmarketIcon iconUrl={invityAPI.getProviderLogoUrl(provider.logo)} />
                     )}
-                    {provider.brandName || provider.companyName}
+                    {provider.brandName ?? provider.companyName}
                 </>
+            ) : (
+                <Row margin={{ left: spacings.xl }}>
```

Comment:
Why the margins on both Row and Text? Why not use the gap on the parent Row?

Reply:
This margin is the same as the width of a missing icon. With a gap, it would not work. I guess.

Reply:
And why not have it in `Text` without `Row`?

Like:

```tsx
<Text margin={{ left: spacings.xxl }}>{exchange}</Text>
```

Reply:
Yes, that's possible!